rfd/0147-repo-mirror.md

```diff
@@ -0,0 +1,73 @@
+---
+authors: Mike Jensen (mike.jensen@goteleport.com)
+state: draft
+---
+
```

Comment:
Who are the intended approvers for the RFD? Could you add the Required Approvers section?

https://github.com/gravitational/teleport/blob/master/rfd/0000-rfds.md#required-approvers

Reply:
I was unsure who needs to approve a RFD like this.  In general there are minimal impacts, and nothing needs to change with `teleport`.  I was just trying to seek broad feedback.

Reply:
As a reviewer I find it useful to know whether I'm an appointed approver, vs just showing up to comment in the few parts I care/know about. As an author, I just point my fingers at whomever I think it appropriate for the section.

Reply:
Required approvers added in 5264dec3627ee1d508eb1d73ae97f735433a75bf